docs/reference/setup/install/docker.asciidoc

```diff
@@ -87,6 +87,11 @@ endif::[]
 This sample Docker Compose file brings up a three-node {es} cluster. 
 Node `es01` listens on `localhost:9200` and `es02` and `es03` talk to `es01` over a Docker network.
 
+Please note that Docker exposes port 9200 to the public, ignoring potential firewall 
```

Comment:
How about:

```
Please note that this configuration exposes port 9200 on all network
interfaces, and given how Docker manipulates `iptables` on Linux, this
means that your {es} cluster is publically accessible, potentially ignoring
any firewall settings.  If you don't want to expose port 9200 and instead
use a reverse proxy, replace `9200:9200` with `127.0.0.1:9200:9200` in the
docker-compose.yml file. {es} will then only be accessible from the host
machine itself.
```

cc @dliappis in case he has any thoughts on the wording here.

@jrodewig is `7.4` the correct target branch for these changes?

Reply:
@Kovah If you could, please target this PR against master. This will ensure we can backport the changes to all maintained branches.

Thanks for pointing that out @pugnascotia.